paymentapi/src/main/java/fi/hel/verkkokauppa/payment/api/PaymentAdminController.java

```diff
@@ -166,4 +171,21 @@ public ResponseEntity<VismaPayResponse> cancelPayment(@RequestParam(value = "pay
         }
     }
 
+    @PostMapping(value = "/payment-admin/online/save-payment-filter", produces = MediaType.APPLICATION_JSON_VALUE)
+    public ResponseEntity<List<PaymentFilterDto>> savePaymentFilter(@RequestBody @Valid List<PaymentFilterDto> paymentFilters) {
+        try {
+            List<PaymentFilter> model = filterService.savePaymentFilters(paymentFilters);
```

Comment:
Hmm, filtersModel & filtersDto might be better since just making words 'model' and 'dto' plural sound a bit odd, as in if there would be several models and data transfer objects. I'll change the naming.

Reply:
But this function handles list of objects so it needs to be prural in my opinion.

function name is savePaymentFilter but you pass list of paymentFilters to function so those does not match

Reply:
I could be renamed to
savePaymentFilterList and function parameter to paymentFilterList

Reply:
In my comment here, I was referring to the fields model and dto - the function name being plural which you also pointed out in another comment below I do agree with and have changed it to be plural to be unified with the parameter.